package.json

```diff
@@ -1,98 +1,28 @@
 {
-  "name": "forklift-console-plugin",
-  "version": "0.0.1",
+  "name": "@monorepo/root",
   "private": true,
+  "version": "0.0.1",
   "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
+  "workspaces": [
+    "./packages/*"
+  ],
   "scripts": {
-    "clean": "rm -rf dist",
-    "build": "yarn clean && NODE_ENV=production yarn webpack",
-    "start": "yarn webpack serve",
+    "start": "npm run start -w @kubev2v/forklift-console-plugin",
+    "build": "NODE_ENV=production npm run build -ws",
+    "clean": "rm -rf dist && rm -rf coverage && npm run clean -ws",
+    "clean:all": "npm run clean && rm -rf node_modules && npm run clean:all -ws",
```

Comment:
1. removing `node_modules` was a surprise - I expected that `all` means all packages
2. in the transition period it makes sense also to remove legacy folders (previously done by `yarn clean`)

Reply:
a. yes, all means all artifacts :-)
2. "clean" now removes the base artifacts like it did before